templates/targets/x86_64/generic/disk-config.nix

```diff
@@ -51,6 +71,95 @@ _: {
               ];
             };
           };
+          vm_storage_a = {
+            size = "30G";
+            content = {
+              type = "filesystem";
+              format = "ext4";
+              mountpoint = "/vmstor";
```

Comment:
Is the missing `e` in the end of the mountpoint intentional or accidental? https://github.com/tiiuae/ghaf/pull/478/files#diff-8a1fbff8658406e95003d28116ba21ced73c81b4fb8d1a5edad5fe5aeef1c427R44

Reply:
Thanks for noticing! Addressed.